libbeat/metric/system/process/process.go

```diff
@@ -217,22 +219,19 @@ func getProcEnv(pid int, out common.MapStr, filter func(v string) bool) error {
 	return nil
 }
 
+// GetProcMemPercentage returns process memory usage as a percent of total memory usage
 func GetProcMemPercentage(proc *Process, totalPhyMem uint64) float64 {
 	// in unit tests, total_phymem is set to a value greater than zero
 	if totalPhyMem == 0 {
-		memStat, err := memory.Get()
-		if err != nil {
-			logp.Warn("Getting memory details: %v", err)
-			return 0
-		}
-		totalPhyMem = memStat.Mem.Total
+		return 0
```

Comment:
why change this? Anything wrong with `memStat.Mem.Total` here?

Reply:
Nevermind, I see the code has been moved. +1 on the change. I think we can also remove the `// in unit tests` comment.